### PR TITLE
Release storageos 1.3.0

### PR DESCRIPTION
**Pull request by Arau**
New kernel modules need testing in the Katacoda images

### CODE REVIEW

File: environments/docker/daemon.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "bip":"172.18.0.1/24",
+    "bip":"1.3.08.0.1/24",
```
Comment (avestuk) [inline environments/docker/daemon.json]: What's this change for?

File: openshift311/env-init.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #ssh root@host01 'sed -i -e "s/slave/rshared/" /etc/systemd/system/multi-user.target.wants/docker.service'
 #ssh root@host01 'systemctl daemon-reload && systemctl restart docker.service'
-ssh root@host01 'curl -skSL https://github.com/storageos/go-cli/releases/download/1.2.0/storageos_linux_amd64 > /usr/local/bin/storageos && chmod +x /usr/local/bin/storageos'
+ssh root@host01 'curl -skSL https://github.com/storageos/go-cli/releases/download/1.3.0/storageos_linux_amd64 > /usr/local/bin/storageos && chmod +x /usr/local/bin/storageos'
```
Comment (avestuk) [inline openshift311/env-init.sh]:
```suggestion
ssh root@host01 'curl -skSL https://github.com/storageos/go-cli/releases/download/1.2.0/storageos_linux_amd64 > /usr/local/bin/storageos && chmod +x /usr/local/bin/storageos'

We should revert this change if the CLI version is not being bumped. 
```

Comment (Arau) [reply openshift311/env-init.sh]: good catch

Comment (Arau) [reply openshift311/env-init.sh]: fixed

File: environments/docker/daemon.json (hunk 1 above)
Comment (Arau) [reply environments/docker/daemon.json]: Don't know. Removing change

**Pull request merged**